project-chip/connectedhomeip
# Enforce length constraints in access control

**Do Not Merge, Let mlepage merge it, needs dependent PR**

#### Problem
Partially fixes #14455
Partially fixes #14460
Partially fixes #16034

#### Change overview
Add some APIs for length constraints. Use them in system module and in
cluster to enforce length constraints.
    
Partially fixes #14455
Partially fixes #14460
Partially fixes #16034

#### Testing
- Build and run (chip-all-clusters-app and chip-tool) on Linux x64
- Verify operation using log statements
- CI
